### ISSUE TITLE
Unable to use the plugin in IntelliJ IDEA 2016.2.2

### ISSUE THREAD

**Issue by czyzby** [gradle/gradle-script-kotlin#115]
When importing the hello world sample, I keep on getting this error:
```
org.gradle.script.lang.kotlin.support.GradleKotlinScriptDependenciesResolver cannot be cast to org.jetbrains.kotlin.script.ScriptDependenciesResolver
```
Needless to say, the project is not properly imported (source folders are not actually marked as source) and `build.gradle.kts` features no code completion.

**Comment by Szil** [comment]
I can confirm it's indeed borked, but its just the Kotlin plugin dies therefore the code completion not working, but i can still build/run Kotlin project with gradle-script-kotlin.

**Comment by czyzby** [comment]
I noticed that the issue is still up (and some people are still facing this problem), so I just wanted to say that I managed to resolve this.

This bug is related to outdated Kotlin plugin linked in some Gradle `README` file, which I manually installed in order to enable autocomplete in IntelliJ. Installing the official early access Kotlin plugin fixes the issue. [Related SO question](https://stackoverflow.com/questions/38966625/configure-intellij-auto-completion-for-gradle-script-in-kotlin).
